Add census to CSV download (fixes #459)

ericboucher opened the pull request.

ericboucher committed:
Add census to CSV download

akegan commented:
looks pretty good. want to wait on QA from vivian to merge.

akegan commented on frontend/src/components/DownloadAnalysisLink/DownloadAnalysisLink.tsx:
```diff
@@ -66,9 +71,31 @@ function onClick(store: Store) {
+      getLegend(method, AdequacyMode.ADEQUATE_60),
+      getLegend(method, AdequacyMode.INADEQUATE)].map(
+        legend => snakeCase([_, group, legend].join('_').replace(
+          '<', 'inf').replace('>', 'sup')
```
what's this?

akegan replied:
maybe lt and gt?

viviandien commented:
Sorry-- many comments. Let me know if you want to discuss @ericboucher 

- remove "ZipCode" and "SpecDesc"
- make all column names uniform in usage of EITHER CamelCase OR snake_case
- to be more descriptive, column names should include ranges and remove special characters
- add "to" to describe ranges instead of special char "< >", abbreviate "years" to "yrs"
- ie: "age_0_to_18_yrs_0_to_30_min", "age_0_to_18_yrs_30_to_45_min" "age_0_to_18_yrs_45_to_60_min", "age_0_to_18_yrs_grt_than_60_min"
- ie: "< 30 min", "< 45 min", "< 60 min", "> 60 min" with "total_0_to_30_min", "total_30_to_45_min", etc.

ericboucher commented:
Addressed most of the comments from @viviandien but also suggested addressing the rest in another PR. Could an engineer re-review this stage? @tetraptych @akegan @philipdickinson ?

akegan commented:
which ones did you address and which ones did you not address?

akegan reviewed (approved):
LGTM but let's make a ticket to speed it up (not through caching)

viviandien commented:
@ericboucher column names still need to be fixed to use "xx_to_yy" instead of "lt_xx" and "gt_yy"
